### ISSUE TITLE
Allow "internal: Run shell path (local)" withouth timeout

### ISSUE THREAD

**Issue by s01itude** [bitfocus/companion#2222]
**Describe the feature**
Allow using the "internal: run shell path (local)" action to be run without a timeout (or add a new action that does the same)

**Usecases**
could be used to launch applications (such as streaming software); currently applications will close once the timeout has been reached

**Comment by Julusian** [comment]
I think some investigation needs to be done to figure out if it is possible to spawn a process that wont be killed when companion exits. And verifying that the same is possible across all platforms

**Comment by Danie10** [comment]
Yes could be useful as I use the run shell path to execute a command to run dates in a terminal screen for my Manjaro Linux. What happens now is the timeout finishes and it just hard closes that terminal screen. The comman I'm running is actually one that is supposed to keep the terminal open even after it ends, but Companion is not allowing it to exist after the timeout expires, so it shuts down prematurely. I'm running:

```
konsole --noclose -e "pamac update --aur --devel"
```

**Comment by Danie10** [comment]
Solved my own problem with this in Linux now by executing the following, which allows the second shell to start up detached from the first, so no matter if the first one is timed out by Companion:

```
bash -c "konsole --noclose -e 'pamac update --aur --devel'" &
```

The & on the end instructs the bash shell to fire up the konsole shell as detached. So I've now got everything working.

**Comment by AJolly** [comment]
Coming from StreamDeck/Windows, this causes me significant heart ache - I did not realize Companion would terminate a process I launched! 

How do I work around this?

**Comment by Danie10** [comment]
That command I gave above - it starts a shell which starts a second shell. When Companion terminates the first shell, the second carries on with the actual commands/processes you wanted to run. Just make sure not to omit the & at the end.

I tried to explain this in my video at https://youtu.be/zakLajaUZYY?si=QEmVJS9TNsaU8lsq&t=2389 - hope it helps make sense of it but it does work.
